Doc/library/threading.rst

```diff
@@ -654,7 +672,7 @@ Reentrant locks also support the :ref:`context management protocol <with-locks>`
 
       Only call this method when the calling thread owns the lock. A
       :exc:`RuntimeError` is raised if this method is called when the lock is
-      unlocked.
+      un-acquired.
```

Comment:
"if this method is called on a non-acquired lock." perhaps?

Reply:
I changed it to just "if this method is called when the lock is not acquired" but we can revise it further if needed.